### DISCUSSION TITLE
Change theme for sql hightlight

### DISCUSSION THREAD

**Discussion by semiherdogan** [spatie/ray#136] [Ideas]
On dark mode sql queries are barely readable. 
![image](https://user-images.githubusercontent.com/5646041/104424988-2a1b8e80-5591-11eb-9204-5e44f4f74efb.png)

On light mode everything is ok.
![image](https://user-images.githubusercontent.com/5646041/104425091-49b2b700-5591-11eb-8423-5fd21a0141b1.png)

Is there any way that we can change theme for sql queries?
Or another idea is adding theme selection.

By the way thank you for this great product, bought yesterday and i don't want to debug anything without it.

**Comment by freekmurze** [comment]
@willemvb could you take a look at this?

**Comment by willemvb** [comment]
I'll create an issue, this shall be fixed soon!

**Comment by AdrianMrn** [comment]
Fixed in 1.2.5 (will release later today)

![image](https://user-images.githubusercontent.com/12762044/104443335-9947b180-5596-11eb-81df-31e4d9f11eb9.png)

**Reply by semiherdogan** [reply]
that was fast 💯 
thank you 👍

**Comment by dansleboby** [comment]
@AdrianMrn I don't have highlight when I view queries, it is normal?
